static/js/zulip.js

```diff
@@ -1,3 +1,7 @@
+// WHEN ADDING NEW GLOBAL VARIABLES, you must add them to
+// the whitelist in tools/jslint/check-all.js in order for
+// tests to pass.
+
```

Comment:
Remove this block since we're actually hoping to stop adding new global variables :)